samples/snippets/src/test/java/com/example/firestore/snippets/QueryDataSnippetsIT.java

```diff
@@ -271,6 +271,12 @@ public void testFilterNotIn() throws Exception {
     assertEquals(expected, actual);
   }
 
+  @Test
+  public void testSnapshotReads() throws Exception {
+    // Verify that this runs without error
+    DocumentSnapshot doc = queryDataSnippets.runStaleReads();
```

Comment:
Since this is a test, consider some assertion here. You may choose not to since that doesn't seem conventional for this file. But it's a good practice to make sure the code actually does what you intend.